mxcube3/core/components/user/usermanager.py

```diff
@@ -385,35 +381,37 @@ def _login(self, login_id: str, password: str):
                 self.force_signout_user(login_id)
             else:
                 if current_user.username == login_id:
-                    raise Exception("You are already logged in here")
+                    raise Exception("You are already logged in")
                 else:
                     raise Exception(
-                        "Login rejected, you are already logged in somewhere else\nand Another user is already logged in here"
+                        "Login rejected, you are already logged in somewhere else\nand Another user is already logged in"
                     )
 
         # Only allow in-house log-in from local host
         if inhouse and not (inhouse and is_local_host()):
             raise Exception("In-house only allowed from localhost")
 
+        non_inhouse_active_users = self.active_logged_in_users(exclude_inhouse=True)
+
         # Only allow other users to log-in if they are from the same proposal
         if (
             (not inhouse)
-            and active_users
-            and (login_id not in active_users)
+            and non_inhouse_active_users
```

Comment:
Just making sure that inhouse users and standard user can be logged-in at the same time